app/views/boards/show.html.erb

```diff
@@ -52,7 +52,7 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= preview_link preview_board_topics_path(@board), 'message-form-preview', class: 'button' %>
       <%= link_to l(:button_cancel), "#", onclick: 'Element.hide("add-message")', class: 'button' %>
     <% end %>
-    <div id="preview" class="wiki"></div>
+    <div id="preview"></div>
```

Comment:
@NobodysNightmare yep, maybe the structure is a bit strange :unamused:  We don't need to apply any styling here though: the `<div>` container is only needed for inserting the Ajax-loaded preview.